packages/create-start/build.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineBuildConfig } from 'unbuild'
+
+export default defineBuildConfig({
+  entries: ['src/index', 'src/cli-entry'],
```

Comment:
why do we need two entry points?

Reply:
This has been chagned to build the files using mkdist to retain the tree structure. This is required for the modules to understand relatively whre their templates are